snapshots/Hub.Operations.json

```diff
@@ -11,8 +11,8 @@
   "remove: partial": "83740",
   "reportDeficit": "121107",
   "restore: full": "86184",
-  "restore: full - with transfer": "197201",
+  "restore: full - with transfer": "207201",
   "restore: partial": "94550",
-  "restore: partial - with transfer": "144241",
+  "restore: partial - with transfer": "146741",
```

Comment:
PR changes does not impact gas costs. This is because of foundry version mismatch.